targets/XPlatCppSdk/source/build/Windows/XPlatCppWindows.vcxproj.ejs

```diff
@@ -76,9 +79,8 @@
     <ProjectReference />
     <ProjectReference />
     <Lib>
-      <AdditionalDependencies>ws2_32.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalLibraryDirectories>
-      </AdditionalLibraryDirectories>
+      <AdditionalDependencies>ws2_32.lib;lib_json.lib;zlib.lib;libssl.lib;libcrypto.lib;wldap32.lib;crypt32.lib;normaliz.lib;libcurl_a_debug.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
So, we've gone from less static dependencies to more.
Is this intended?
Is this good?

Reply:
This is unfortunate. Because modern versions of libcurl and its dependencies seem to require that. Each one of these dependencies was painfully found when troubleshooting linker errors and following symbol relationships between libs with utilities like DUMPBIN and alike.

---
In reply to: [240441689](https://github.com/PlayFab/SDKGenerator/pull/357#discussion_r240441689) [](ancestors = 240441689)